pkgs/development/compilers/urweb/default.nix

```diff
@@ -4,11 +4,11 @@
 
 stdenv.mkDerivation rec {
   name = "urweb-${version}";
-  version = "20170105";
+  version = "20170720";
 
   src = fetchurl {
     url = "http://www.impredicative.com/ur/${name}.tgz";
-    sha256 = "2ad3aea2c4851c9b18f752d38c7127af8293fbbbbdb3dd06b73a4603fe399b67";
+    sha256 = "17qh9mcmlhbv6r52yij8l9ik7j7x6x7c09lf6pznnbdh4sf8p5wb";
```

Comment:
Huh - I'm slightly confused that the sha256 is short, but it works.  (I got this using my usual hack of attempting to build it and copying it from the "wrong hash" error message.)